docs/pages/application-access/cloud-apis/google-cloud.mdx

```diff
@@ -0,0 +1,615 @@
+---
+title: "Protect Google Cloud CLIs with Teleport"
+description: How to enable secure access to Google Cloud CLIs.
```

Comment:
In fact the Google Cloud Platform is accessed via google CLI 

so maybe instead
> Protect Google Cloud CLIs with Teleport

the title should sound something like: "Protect Google Cloud Platform Access with Teleport"